lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -273,7 +278,28 @@ const Reportback = {
         }
       },
     }).done(function(response) {
-      _this.apiUrl = response.meta.pagination.links.next_uri || null;
+      // If response.meta.pagination.links.next, this is from Rogue.
+      if (response.meta.pagination.links.next) {
+        var splitResponseUrl = response.meta.pagination.links.next.split("&");
```

Comment:
Ooh that makes sense! Totally down to try that but just thinking out loud here - is there any benefit to having just `next` over `next_uri` (is that a good standard that apps will you across the board?). Wondering because I know we chatted about not worrying about how code is written since we'll trash Phoenix-ashes soon and wondering if it's better code for Rogue to have `next` instead of `next_uri`?